SampleApp/SearchPinsViewController.swift

```diff
@@ -9,16 +9,23 @@
 import Pelias
 import TangramMap
 
-class SearchPinsViewController: SampleMapViewController, UITextFieldDelegate {
+class SearchPinsViewController: SampleMapViewController, UITextFieldDelegate, AutocompleteSearchDelegate {
 
   @IBOutlet weak var searchField: UITextField!
 
+  @IBOutlet weak var displaySearch: UIButton!
+  let searchListSegueId = "searchListSegueId"
+
   override func viewDidLoad() {
     super.viewDidLoad()
 
-    try? loadStyle(.bubbleWrap)
+    try? loadStyleAsync(.bubbleWrap) { [unowned self] (style) in
```

Comment:
Do you have any idea why the async methods would show a black screen while attempting to load the scene?